src/router/index.js

```diff
@@ -43,6 +51,11 @@ export default function ({ store }) {
             next()
           }
         } else {
+          // To fix scrolling top when changing query params
+          const element = document.getElementById('multi-dho-scroll-area').children[0]
```

Comment:
i suggest you use [quasar scroll utils here also ](https://quasar.dev/quasar-utils/scrolling-utils)

Reply:
@arsenijesavic same situation as in your first comment